docs/developers/guides/automatic_coin_conversion.md

```diff
@@ -0,0 +1,81 @@
+<!--
+order: 8
+-->
+
+# Automated Coin Conversion
+
+Learn how to the Automated Coin Conversion feature works. {synopsis}
+
+In their ERC-20 representation, assets can be used to interact with dApps using the EVM. This standard allows developers to build applications that are interoperable with other products and services. In their Native Coin representation, they can be transferred between accounts on Evmos and other Cosmos chains using IBC. They cannot, however, be used to interact with dApps on Evmos, as Native Coins are not supported by the EVM since they don’t implement the ERC-20 standard.
```

Comment:
Small nit, I think singular 'representation' sounds like its ONE representation for ALL of the assets, whilst these are many different ERC-20 representations.

Also, I think adding some links here (like for the ERC-20 standard) would be helpful as well :)
```suggestion
In their [ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) representations, assets can be used to interact with dApps using the EVM. This standard allows developers to build applications that are interoperable with other products and services. In their Native Coin representation, they can be transferred between accounts on Evmos and other Cosmos chains using IBC. They cannot, however, be used to interact with dApps on Evmos, as Native Coins are not supported by the EVM since they don’t implement the ERC-20 standard.
```